.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1497,10 +1484,13 @@ public class ChatToolCall : IJsonModel<ChatToolCall>, IPersistableModel<ChatTool
         public override readonly string ToString();
     }
     public class ChatToolChoice : IJsonModel<ChatToolChoice>, IPersistableModel<ChatToolChoice> {
-        public ChatToolChoice(ChatTool tool);
         public static ChatToolChoice Auto { get; }
         public static ChatToolChoice None { get; }
         public static ChatToolChoice Required { get; }
+        public static ChatToolChoice CreateAutoChoice();
```

Comment:
We should explicitly review this with @KrzysztofCwalina -- I know he was opposed to the "just include both properties and methods" approach we took to expedite `response_format` and this is where we should formalize and standardize what we want to do about these everywhere.

Reply:
I'm personally in favor of keeping only the factory methods. If for some reason we need to pivot back to properties in the future, we can always add them back later.